src/reynir_correct/wrappers.py

```diff
@@ -177,16 +171,39 @@ def val(t: CorrectToken, quote_word: bool = False) -> Union[None, str, float, Tu
 @dataclass
 class CorrectedSentence:
     tokens: List[CorrectToken]
+    parsed: bool
     annotations: Optional[List[Annotation]] = None
-    terminals: Optional[List[Terminal]] = None
 
-    def filter_annotations(self, ignore_rules: Set[str]):
+    @staticmethod
+    def from_parser_sentence(sentence: AnnotatedSentence) -> "CorrectedSentence":
+        """Create a CorrectedSentence from a parser sentence"""
+        tokens = sentence.correct_tokens
```

Comment:
I decided to do it in the constructor since the tokens returned are actually "CorrectToken" and I could not set it since the `tokens` is a @property. I thought this was the most understandable and introduced potentially fewer casts.